.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = tab
+indent_size = 1
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Hm, TBH, I am not a Github expert, and I don't know how to take a commit out of an existing PR. If you don't like the additional .editorconfig, I recommend that you just delete it afterwards. It was just for the sake of convenience, to make life easier for upcoming contributors.